dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/dataAnalysis/CommandStateCountResponse.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.api.dto.dataAnalysis;
+
+import org.apache.dolphinscheduler.api.enums.Status;
+import org.apache.dolphinscheduler.api.utils.Result;
+import org.apache.dolphinscheduler.common.Constants;
+import org.apache.dolphinscheduler.dao.entity.Project;
+
+import java.util.Map;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * command state count response
+ */
+@Data
+@NoArgsConstructor
+public class CommandStateCountResponse extends Result {
+
+    private Project data;
+
+
+    public CommandStateCountResponse(Map<String, Object> result) {
```

Comment:
Please replace `Map` to pojo since this is our expectation of reconstruction.

Reply:
CommandStateCountResponse(Map<String, Object> result) => CommandStateCountResponse(Result result)

Reply:
You can take a look at `org.apache.dolphinscheduler.api.controller.AccessTokenV2Controller#createToken`